### PR TITLE
Replacing the two arrows with 'back' and 'read more'

### PR DESCRIPTION
**Pull request by g1039**
@miltontony please review (https://praekeltorg.atlassian.net/browse/TUNE-63) cc @hodgestar

### CODE REVIEW

**Comment by miltontony** [comment]
Please hold off on this - I'm following up with the TuneMe team - there's some discrepancy on what the final implementation should be.

**Comment by g1039** [comment]
@hodgestar @saeedmb8 Please review

**Comment by g1039** [comment]
Ready for re-review

File: tuneme/templates/comments/comments.html [hunk 1]
```diff
@@ -18,13 +18,13 @@
 <div class="pagination">
     <span class="step-links">
         {% if page.has_previous %}
-            <a href="?p={{ page.previous_page_number }}">&larr;</a>
+            <a href="?p={{ page.previous_page_number }}">&lt; &ndash; &ndash; &ndash;</a>
         {% endif %}
```
Comment (smarzban) [inline tuneme/templates/comments/comments.html]: @TrevorOctober Can you please remove the space between dashes, it should be something like `&lt;&ndash;&ndash;&ndash;`

File: tuneme/templates/comments/comments.html [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
         <span class="current">
             Page {{ page.number }} of {{ page.paginator.num_pages }}
         </span>
         {% if page.has_next %}
-            <a href="?p={{ page.next_page_number }}">&rarr;</a>
+            <a href="?p={{ page.next_page_number }}">&ndash; &ndash; &ndash; &gt;</a>
```
Comment (smarzban) [inline tuneme/templates/comments/comments.html]: @TrevorOctober here as well `&ndash;&ndash;&ndash;&gt;`

**Comment by g1039** [comment]
@saeedmb8 Please review

**Comment by smarzban** [comment]
@TrevorOctober please make it to be centre as well

**Comment by g1039** [comment]
@saeedmb8 Please review again

File: tuneme/static/css/styles.css [hunk 3]
```diff
@@ -183,7 +183,7 @@ label {font:normal 15px 'Quicksand', Arial, Helvetica, sans-serif;}
         .magenta .article-copy blockquote, .comment.magenta .by strong, .comment.magenta .report {color:#e60382;}
 
 
-.pagination {padding:15px; text-align: left}
+.pagination {padding:15px; text-align: center;}
```
Comment (smarzban) [inline tuneme/static/css/styles.css:189]: @TrevorOctober this will centre the whole pagination, you just want to centre `<---  page 2 of 3 --->`

**Comment by g1039** [comment]
@saeedmb8 ^^^

**Comment by smarzban** [comment]
@TrevorOctober can you add a screenshot

**Comment by smarzban** [comment]
@TrevorOctober review please

**Comment by g1039** [comment]
Looks good :+1:

**Pull request merged**